lib/bindConnectors/bindDynamicOutput.js

```diff
@@ -0,0 +1,31 @@
+var _                = require('lodash');
+var when             = require('when');
+var GenerateSchema = require('generate-schema');
+
+module.exports = function (message, options) {
+
+    return function (event) {
+
+        return when.promise(function (resolve, reject) {
+
+            when(message.dynamicOutput(event.body.input))
```

Comment:
Unless I've got it wrong, I believe `input` is the same as `params` when you're working in the `model.js`.
Why should there be more data included?

@chrisjhoughton probably has a better idea of what `event` object has. I just followed the example from one of the other files, like in `bindTriggerRequest.js`.